lib/src/doc/lives.rs

```diff
@@ -100,42 +96,139 @@ impl<'a> Document<'a> {
 				// Finally, let's check what type of statement
 				// caused this LIVE query to run, and send the
 				// relevant notification based on the statement.
+				let mut tx = txn.lock().await;
+				let ts = tx.clock().await;
+				let ns = Box::new(opt.ns().to_string());
+				let db = Box::new(opt.db().to_string());
+				let tb = Box::new(self.id.unwrap().tb.to_string());
+				let not_id = crate::sql::Uuid::new_v4();
 				if stm.is_delete() {
```

Comment:
Small nit pick, may be a match here helps improve readability? WDYT?

```
    let notification = match stm {
        Statement::Delete => {
            // Logic for DELETE notification
            Notification {
            }
        }
        Statement::Update => {
            // Logic for UPDATE notification
            Notification {
            }
        }
    };
```

Reply:
We tried this with live queries before 1.0, and there were a multitude of issues
- stm isn't the live query, it is the statement creating data for the live query. So we need to check properties, not statements
- borrow is a pain
- delete (notifies of previous value before deletion) operates differently to create and udpate (notifies of value after create/update)

So it's a good suggestion, but it is difficult to achieve in this case unfortunately. We could do it as a further improvement.